packages/core/src/Account.ts

```diff
@@ -19,6 +19,7 @@ export type MnemonicAccount = {
   seedFingerPrint: string;
   address: ImplicitAddress;
   pk: string;
+  isVerified: boolean;
```

Comment:
I think this one is a bit problematic to handle because there can be only 1 unverified account anyway, but this schema allows for multiple.
what if we just add a new key to the accountsSlice itself? this way we can have only 2 states - either verified or not verified among all accounts so to speak.

then the useIsVerified simplifies to just 

```typescript
const useIsVerified = () => useAppSelector(s => s.accounts.isVerified)
```

instead of 
```typescript
const accountIsVerified = accounts.every(acc => acc.type === "mnemonic" && acc.isVerified);
```

also, with this implementation you forgot to update `restoreFromMnemonic`
whilst with the "global" `isVerified` it would've been something like:
```typescript
if (mode === 'new_mnemonic') {
  dispatch(accountActions.setVerified(false);
}
```
in the SetupPassword modal